packages/pyolite-kernel/py/ipykernel/ipykernel/comm.py

```diff
@@ -41,12 +41,17 @@ def _publish_msg(self, msg_type, data=None, metadata=None, buffers=None, **keys)
         metadata = {} if metadata is None else metadata
         content = dict(data=data, comm_id=self.comm_id, **keys)
 
+        bytes_buffers = []
+        if buffers:
+            for each_buffer in buffers:
+                bytes_buffers.append(each_buffer.tobytes())
+
```

Comment:
Perhaps more similar to to the other default handlers might be...

```py
buffers = None if buffers is None else [b.to_bytes() for b in buffers]
```

and wouldn't require the change below, which is a hair confusing...

Reply:
Thanks @bollwyvl, just pushed again. I will wait for the RTD to check it again though. Since I am not able to check it locally because it picks up cached contents.

Reply:
So many caches. What's the _particular_ cache situation?  Seems like we should fix it for local dev, if it's causing you issues... i guess issue up if this is a persistent (ha!).